docs/translating/polish.md

```diff
@@ -74,10 +78,46 @@ Zulip friendly and usable.
   (za)subskrybować kanał (verb), perfective form depending on the
   context
 
-**topic**: wątek
+**topic**: wątek example:Add new topic - Dodaj nowy wątek
 
 **unsubscribing from a stream**: odsubskrybowanie kanału (noun) and odsubskrybować kanał (verb)
 
 **view**: widok
 
-**starred message**: oznaczona wiadomość
+**starred message**: oznaczona wiadomość (noun) and oznacz wiadomość (verb)
+
+**message**: wiadomość example:You have got new message - Masz nową wiadomość
+
+**public stream**: kanał publiczny
+
+**bot**: bot
+
+**Integration**: integracja
+
+**Notification**: Powiadomienie example: Turn on notification - Włącz powiadomienia
+
+**alert word**: ostrzeżenie
+
+**narrow to**: zwięź do
+
+**mute**: wycisz
+
+**unmute**: wyłącz wyciszenie
+
+**deactivate**: dezaktywować
+
+**reactivate**: reaktywować
+
+**search**: znajdź 
+
+**pin**: przypnij or oznacz
+
+**invalid**: niepoprawny/a example:Command is invalid - Komenda jest niepoprawna
+
+**i want**: chcę example: I want to change password. - Chcę zmienić hasło
+
+**user**: użytkownik example:Add new user - Dodaj nowego użytkownika
+
+**person**: osoba
+
+**people**: ludzie
```

Comment:
here, I'm not clear on the usage of "people" -- in general I'd avoid it and write "users", as it sounds a tad stiff to use "people"